docs/pages/enroll-resources/desktop-access/troubleshooting.mdx

```diff
@@ -393,3 +393,34 @@ to force the policy update:
 ```code
 $ gpupdate.exe /force
 ```
+### Shared Directory Error
+
+Attempts to share directory on a remote desktop session results in the error:
+
+```
+Encountered shared directory error 1
+```
+
+**Solution:** Review and disable configured Device Redirection Group Policy
+
+First, make sure that all [Directory sharing](https://goteleport.com/docs/enroll-resources/desktop-access/directory-sharing/#prerequisites) prerequisites has been met.
+
+Teleport requires device redirection policies to be disabled in RDP policies to allow directory sharing from a remote computer in an RDP session
+
+To fix this issue, review and disable the `Device and resource redirection policy` within the `Remote Desktop session host` path below:
+
+```text
+Computer configuration > Policies > Administrative Templates > Windows Components > Remote Desktop Services > Remote Desktop Session Host > Device and Resource Redirection > *
+```
+
+Some key policies to lookout for:
+
+- Do not allow COM port redirection
+- Do not allow drive redirection
+- Do not allow LPT port redirection
```

Comment:
I wouldn't expect COM port redirection or LPT port redirection to matter for directory sharing.

In fact, I can disable these and directory sharing still works as expected:

<img width="1067" alt="image" src="https://github.com/user-attachments/assets/e631988d-9ecd-4cd1-bdbc-7f37984d19b5">

As soon as I disable device redirection I do get the original error, so I think you can remove these two.